data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,70 @@
+import requests
+import os
+import psycopg2
+import getpass
+import logging
+
+# set up logging
+logging.basicConfig(format='%(asctime)s - %(levelname)s @' +
+                    '%(module)s-%(funcName)s : %(message)s',
+                    level=logging.INFO)
+
+# for connecting to remote database
+# conn = psycopg2.connect(database='yourdb', user='dbuser', password='abcd1234', host='server', port='5432', sslmode='require')
+# for creating local database initially
+# con = psycopg2.connect(dbname='postgres', user=getpass.getuser(), host='localhost', password='Pa55word')
+# cur = con.cursor()
+# cur.execute("CREATE TABLE Jobs(Id SERIAL PRIMARY KEY, job_title TEXT, link TEXT, description TEXT, city TEXT, state TEXT, country TEXT, latitude FLOAT, longitude FLOAT, company_name TEXT, salary_min INTEGER, start_date DATE)")
+# con.commit()
+# cur.close()
+# con.close()
+
+def request_ziprecruiter_jobs(page_num, api_key):
+  url = "https://api.ziprecruiter.com/jobs/v1?search=internship%20Job&days_ago=1&page={page_num}&jobs_per_page=20&api_key={api_key}" \
+  .format(page_num=page_num, api_key=api_key)
+  req = requests.get(url)
+  response = req.json()
+  if response["success"]:
+    con = psycopg2.connect(dbname='postgres', user=getpass.getuser(), host='localhost', password='Pa55word')
```

Comment:
Passwords? In _my_ plaintext? I suppose it's not a huge deal, but that field should probably be read in from an environment variable.